docs/source/release/v6.3.0/muon.rst

```diff
@@ -85,35 +75,32 @@ Improvements
 
 - :ref:`LoadPSIMuonBin <algm-LoadPSIMuonBin>` can now load a subset of the spectra.
 
-Bugfixes
-########
-
 
 Fitting Functions
 -----------------
 New Features
 ############
-* Added two Activation fitting functions to MuonModelling Fit Functions.
+* Added two Activation fitting functions to ``MuonModelling`` Fit Functions.
 
-  * `ActivationK <func-ActivationK>` can be used for data in Kelvins.
-  * `ActivationmeV <func-ActivationmeV>`can be used for data in meV.
+  * :ref:`ActivationK <func-ActivationK>` can be used for data in Kelvins.
+  * :ref:`ActivationmeV <func-ActivationmeV>`can be used for data in meV.
```

Comment:
This link does not work

Reply:
Is it both links that don't work or just the second one?